.babelrc

```diff
@@ -1,9 +1,18 @@
 {
-  "plugins": [
-    "emotion",
-    "transform-class-properties",
-    "transform-object-rest-spread"
+  "plugins": ["emotion", "@babel/plugin-proposal-class-properties"],
+  "presets": [
+    ["@babel/preset-env", { "modules": false }],
```

Comment:
Is there a reason for changing this to `{modules: false}`  here?

For context,  our build script uses rollup to compile to UMD and ESM, we use the babel-cli to build our lib folder as CJS. the implicit `{ modules: auto }` accommodates this behaviour see [here](https://github.com/babel/babel/blob/master/packages/babel-preset-env/src/normalize-options.js#L150)

Reply:
`modules: false` will be passed by rollup-plugin-babel. You may not specify it. And remove the following `env`